packages/component/src/Localization/en-US.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Can we alphabetize these groups?

If not, when we have to add new strings, should we add them to the bottom?

Reply:
* new strings that don't already have a group

Reply:
The default sorter doesn't work with comments because they starts with `_` (underscore).

Will manual sort it for now.